export const GetAll = async( )=>{
    try {
        
    } catch (err) {
        
    }
}